### ISSUE TITLE
Soft and/or hard limit to number of recipients

### ISSUE THREAD

**Issue by marclaporte** [jasonmunro/cypht#531]
Every once in a while, I'll receive an email with 50+ recipients. And with more recipients, there is a good chance someone will reply to all.

It is usually an error.

Many clients and email services have limits. Ex.: 

- https://github.com/roundcube/roundcubemail/issues/5132
- https://docs.gandi.net/en/gandimail/limitations.html

We should have warnings to help users.

**Comment by dumblob** [comment]
> We should have warnings to help users.

Yep, persistent (i.e. visible during the whole compose session - even if opened from a draft folder) **warning**. But please no hard **limit**.

**Comment by jasonmunro** [comment]
@marclaporte to clarify, you want to have a warning if you reply-all to a message with > X recipients. That is not what the PR does so I just want to clarify before I comment on it. The PR uses JavaScript to trigger the warning on a keyup event, so it would only trigger if you are actually typing in the recipients. Also the way it is coded the message cannot be translated and it's possible it will miscount as recipient lines are notoriously hard to parse properly.

**Comment by marclaporte** [comment]
You understood correctly. But basically it doesn't matter if it's typed in or reply to all, there should be a soft warning at one point.

**Comment by jasonmunro** [comment]
I don't think a warning about typing in 20 recipients makes a lot of sense, but we can add that as well if you think it does. I posted some code to the PR that will warn if you load a draft or reply to a message that has > 20 recipients from the to + cc line. Often larger recipient lists are in the bcc field so I chose not to include that toward the warning

**Comment by marclaporte** [comment]
Your proposal works for me :-)

**Comment by marclaporte** [comment]
For the record: https://github.com/cypht-org/cypht/commit/293d1af576d7483145c87d75be65fbefbbc9450e
